api/src/main/java/org/hawkular/inventory/api/model/MetricType.java

```diff
@@ -157,7 +155,7 @@ private Update() {
 
         public Update(Map<String, Object> properties, MetricUnit unit) {
             super(properties);
-            this.unit = unit;
+            this.unit = unit == null ? MetricUnit.NONE : unit;
```

Comment:
This would prevent an update object that would only update properties. See line 72 and the use of `valueOrDefault()` method.

While metric type unit is currently the only "attribute" behaving that way, the resource type version used to behave like that in the past, too, and I think we should keep that. If a value of some attribute is not specified in the update object, the value from the original entity should be used.

Actually, I am not sure why the "properties" don't seem to behave that way, too. That is probably a bug, but that should be handled separately.